app/src/main/java/com/starry/greenstash/ui/navigation/Screens.kt

```diff
@@ -1,69 +1,36 @@
-/**
```

Comment:
I think you mistakenly removed the header comment from this file. Please make sure to add it back after making the other requested changes. Thanks!

Reply:
Done